File: NewsBubble/Assets/Scripts/voteArticle.js
#pragma strict
var likeArticle : int;

function Start () {
	if(PlayerPrefs.GetInt("voteArticle") == null){
			
			PlayerPrefs.SetInt("voteArticle", 0);
	}		
	likeArticle = PlayerPrefs.GetInt("voteArticle");
}

function OnMouseDown () {
	likeArticle++;
	PlayerPrefs.SetInt("voteArticle", likeArticle);
}

function OnGUI () {
	GUI.Label(Rect(540,50,160,40), "Like: " + PlayerPrefs.GetInt("voteArticle"));
	GUI.Label(Rect(1180,50,160,40), "Like: " + PlayerPrefs.GetInt("voteArticle"));
}

File: NewsBubble/Assets/Scripts/voteArticle.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class voteArticle : MonoBehaviour {

	public Text LikeCounter;
	int likes = 0;
	
	// Use this for initialization
	void Start () {
		if(PlayerPrefs.GetInt("voteArticle") == null){
			
			PlayerPrefs.SetInt("voteArticle", 0);
		}		
		likes = PlayerPrefs.GetInt("voteArticle");
		LikeCounter.text = likes.ToString ();
	}
	
	// Update is called once per frame
	void OnMouseDown () {
		likes++;
		PlayerPrefs.SetInt("voteArticle", likes);
		LikeCounter.text = likes.ToString();
	}
}
